tools/cldr-apps/js/webpack.config.js

```diff
@@ -12,7 +32,7 @@ module.exports = (env, argv) => {
       cacheDirectory: path.resolve(__dirname, '../target/webpack_cache'),
     },
     output: {
-      filename: "bundle.js",
+      filename: "[name].[contenthash].js",
```

Comment:
Reference for "[name]": https://webpack.js.org/guides/output-management/

Reference for "[contenthash]": https://webpack.js.org/guides/caching/

I wish I understood better. Is "[name]" going to get replaced by "index", based on "entry: "./src/index.js"?

Reply:
hmmm... so where would "main" come from in this case, and where's the code change that results in loading pieces dynamically?

Reply:
Main is the name of the segment in webpack… We could put all of the vendor code, such as vue.js into a separate segments, since it won't change often. 

The .js I added writes the segment list to a new manifest.json file and surveytool.Java reads it. 

When we have a static index.html then webpack will take care of embedding the bundle automatically.

Reply:
Actually the docs are here https://webpack.js.org/configuration/output/#template-strings

Reply:
that's helpful! also here's a glossary: https://webpack.js.org/glossary/

defines, e.g., "chunk" :-)